doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md

```diff
@@ -0,0 +1,353 @@
+    "pluginTypeName":"Cassandra",
+    "tenantId":"1"
+}
+```
+
+# Playing with Cassandra log appender
+
+We'll use [Data collection demo](https://github.com/kaaproject/sample-apps/tree/master/datacollectiondemo/source) from Kaa Sandbox. Our example will send data
+to Kaa and then persist it to Cassandra. Also, we'll do selection queries on persisted data.
+
+We have next log schema:
+
+```json
+{
+    "type":"record",
+    "name":"LogData",
+    "namespace":"org.kaaproject.kaa.schema.sample.logging",
+    "fields":[
+        {
+            "name":"level",
+            "type":{
+                "type":"enum",
+                "name":"Level",
+                "symbols":[
+                    "KAA_DEBUG",
+                    "KAA_ERROR",
+                    "KAA_FATAL",
+                    "KAA_INFO",
+                    "KAA_TRACE",
+                    "KAA_WARN"
+                ]
+            }
+        },
+        {
+            "name":"tag",
+            "type":"string"
+        },
+        {
+            "name":"message",
+            "type":"string"
+        },
+        {
+            "name":"timeStamp",
+            "type":"long"
+        }
+    ]
+}
+```
+
+The following JSON example matches the previous schema.
+
+```json
+{
+    "level":"KAA_INFO",
+    "tag":"TEST_TAG",
+    "message":"My simple message",
+    "timeStamp":"1466075369795"
+}
+```
+
+Go to Data collection demos in Sandbox.
+
+![Data collection demo in Sandbox](attach/data-collection-demo-in-sandbox.png)
+
+Follow **Installation** instructions.
+
+Next, in the Admin UI follow to **Data collection demo** application.
+
+Go to application's **Log appenders** configuration and add a new one.
+
+![Add log appender](attach/data-collection-demo-application.png)
+
+Enter name of the new appender (we'll use "Cassandra")
+
+Add application token and Timestamp as Log metadata.
+
+Select **Cassandra** appender type.
+
+![Select Cassandra appender type](attach/cassandra-appender-type.png)
+
+Add new node in the **Configuration** section (localhost:9042)
+
+![Add new node](attach/configuration-section.png)
+
+Add auth details if needed (for Sandbox it's empty)
+
+Fill keyspace name. "kaa" is used in this example, because it's already created on a Sandbox machine.
+
+"logs_example" is used as the **Table name pattern**.
+
+![Keyspace configuration](attach/configuration-section-keyspace.png)
+
+The important part of configuration is **Column Mapping**:
+
+![Column mapping](attach/column-mapping.png)
+
+Other configuration:
+
+![Other configuration](attach/other-configuration.png)
+
+Now click **Add** button on the top of the screen to create and deploy appender.
+
+![Add button](attach/add-button.png)
+
+Verify that newly created appender has appeared in list.
+
+![Verify newly created appender](attach/verify-created-appender.png)
+
+Now run Data collection demo application:
+
+```bash
```

Comment:
Could this be split to command and a separate part (with short description) of the output?